### PR TITLE
Fix SC2039

### PR DESCRIPTION
**Pull request by darko-poljak**
SC2039

In POSIX sh, something is undefined.

https://github.com/koalaman/shellcheck/wiki/SC2039

__package_pkg_freebsd I left untouched because this type is meant to be run on FreeBSD anyway and in its shell those are defined.
Also, in type files scripts, where shebang is bash I left non-POSIX _something_ unchanged.

### CODE REVIEW

File: cdist/conf/type/__docker/manifest [hunk 1]
```diff
@@ -24,7 +24,7 @@ state=$(cat "$__object/parameter/state")
 
 case "$os" in
     centos)
-        if (source "$__global/explorer/os_release" && [ "${VERSION_ID}" = "7" ]); then
+        if (. "$__global/explorer/os_release" && [ "${VERSION_ID}" = "7" ]); then
```
Comment (asteven) [inline cdist/conf/type/__docker/manifest]: $__global/explorer/os_release is supposed to executed, not sourced.

Comment (darko-poljak) [reply cdist/conf/type/__docker/manifest]: @asteven Aren't **source** and **.**(dot) the same and only dot is defined by POSIX?

Comment (asteven) [reply cdist/conf/type/__docker/manifest]: They are. But explorers are **executables**. There is no need to source them at all.

Comment (darko-poljak) [reply cdist/conf/type/__docker/manifest]: @asteven  As I understand in the above code "$__global/explorer/os_release" is the output of os_release explorer after it has been run on target host. This output is read and executed using *dot* in the current env and then it is checked if VERSION_ID equals 7. I understand that the output of os_release contains such shell compatible variable definition.

Comment (darko-poljak) [reply cdist/conf/type/__docker/manifest]: @asteven Ping

**Pull request merged**